Makefile

```diff
@@ -21,12 +21,12 @@ $(eval $(call make-global-rule,distclean,$(languages)))
 $(eval $(call make-global-rule,install,$(languages)))
 
 #
-# Install documentation and slice files (don't install IceDiscovery and IceLocatorDiscovery slice files)
+# Install documentation and slice files (don't install IceDiscovery and IceLocatorDiscovery Lookup slice files)
```

Comment:
Renamed Ice[Locator]Discovery.ice to Lookup.ice to avoid conflict with the new header files.